tests/checks/remotesettings/test_certificates_expiration.py

```diff
@@ -25,6 +26,25 @@
 3oC6yXiIDAIwSy4Eylf+/nSMA73DUclcCjZc2yfRYIogII+krXBxoLkbPJcGaitx
 qvRy6gQ1oC/z
 -----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+MIIDBjCCAougAwIBAgIIFl9i2vJHvFQwCgYIKoZIzj0EAwMwgaMxCzAJBgNVBAYT
+AlVTMRwwGgYDVQQKExNNb3ppbGxhIENvcnBvcmF0aW9uMS8wLQYDVQQLEyZNb3pp
+bGxhIEFNTyBQcm9kdWN0aW9uIFNpZ25pbmcgU2VydmljZTFFMEMGA1UEAww8Q29u
+dGVudCBTaWduaW5nIEludGVybWVkaWF0ZS9lbWFpbEFkZHJlc3M9Zm94c2VjQG1v
+emlsbGEuY29tMB4XDTIxMDEwMTE3NTgwNFoXDTIxMDMyMjE3NTgwNFowgakxCzAJ
+BgNVBAYTAlVTMRMwEQYDVQQIEwpDYWxpZm9ybmlhMRYwFAYDVQQHEw1Nb3VudGFp
+biBWaWV3MRwwGgYDVQQKExNNb3ppbGxhIENvcnBvcmF0aW9uMRcwFQYDVQQLEw5D
+bG91ZCBTZXJ2aWNlczE2MDQGA1UEAxMtcGlubmluZy1wcmVsb2FkLmNvbnRlbnQt
+c2lnbmF0dXJlLm1vemlsbGEub3JnMHYwEAYHKoZIzj0CAQYFK4EEACIDYgAE9ft2
+ReAOA01a9iy9RCkXN3v8+6ti5Ba8pjqjq0JYy0D0CgOWFGMWqCCqgPGNC4Ydz1C1
+Ssdh0cjtMHqhGPSpGFttTZqQ+tHa5vfwo7n+KDRqfiY3P4dv/PJitbgzA0hIo4GD
+MIGAMA4GA1UdDwEB/wQEAwIHgDATBgNVHSUEDDAKBggrBgEFBQcDAzAfBgNVHSME
+GDAWgBSgHUoXT4zCKzVF8WPx2nBwp8744TA4BgNVHREEMTAvgi1waW5uaW5nLXBy
+ZWxvYWQuY29udGVudC1zaWduYXR1cmUubW96aWxsYS5vcmcwCgYIKoZIzj0EAwMD
+aQAwZgIxAJFusKXlUSnY8n5ldvqMBUPXZFh0oAitBu1sxpZNGpNuEBSuEWlsAQDE
+gbUgGG/FqgIxANT9km7V/qkriJ99c2MIZsPhKh7suUoPjd7XpbXsRLNle9hxop92
+9eHUYBDGIH8/iA==
+-----END CERTIFICATE-----
```

Comment:
This second cert expires after the first one. The datetime shown in the error message in the test below shows that it checks for the one that expires earliest